src/presenter/trip-presenter.js

```diff
@@ -149,6 +149,11 @@ export default class TripPresenter {
         this.#removeLoading();
         this.#render();
         break;
+      case UpdateType.INIT_ERROR:
+        this.#isLoading = false;
+        this.#removeLoading();
+        render(new MessageView({ message: MessageType.FAILEAD }), this.#tripEventsElement); // удаление и #isFailedLoading нет т.к. только возможно нажать обновить страницу
```

Comment:
И константу надо обновить

```suggestion
        render(new MessageView({ message: MessageType.FAILED }), this.#tripEventsElement); // удаление и #isFailedLoading нет т.к. только возможно нажать обновить страницу
```

Reply:
переделал на Model.init.catch